### ISSUE TITLE
[details/header]: Fix playtime heatmap

### ISSUE THREAD

**Issue by pinpox** [megaclan3000/megaclan3000#86]
![image](https://user-images.githubusercontent.com/1719781/85126198-45a5fd00-b22d-11ea-8ae4-ffbb5ccd194e.png)
The heatmap shows the playtime in the last two weeks. It is filled with test data for now.
- [ ] Use real data
- [x] Style to match other UI elements

**Comment by moritzpflueger** [comment]
### how to use real data?
1. i only found `Playtime2Weeks` which returns the time a player was online the last 2 weeks
2. is there data for the whole year? or was this simply in the mockup
3. *how* can i access this data in the html-template? `{{.PlayerHistoryEntry}}` doesn't seem to exist

PS: until now I've only used `UserStatsForGame` and I'm still fairly new to the project. I can't find where the API Data is stored 🤒

**Comment by pinpox** [comment]
> PS: until now I've only used `UserStatsForGame` and I'm still fairly new to the project. I can't find where the API Data is stored

No worries, this one is a little complicated, let me explain:
The API does only provide momentary values as e.g. "Number of Headshots", "Number of rounds played on Map X" or "Playtime in the last 2 Weeks". There is no history of values at all.

To work around this limitation of the API, I added an additional table to the backend, which is represented by the [PlayerHistory](https://godoc.org/github.com/megaclan3000/megaclan3000/internal/steamclient#PlayerHistory) object in code. As you can see in the link above, the `PlayerHistory` object just holds an `SteamID` and an array of [PlayerhistoryEntrys](https://godoc.org/github.com/megaclan3000/megaclan3000/internal/steamclient#PlayerHistoryEntry) called `Data`. These contain some, but not all of the fields from other objects and a field called `Time`. 

The `Time` is when this "snapshot" of the values where taken. 

TL;DR: You can loop over the `.PlayerHistory.Data` array to get a history of a specific value like this:

```html
Showing data for {{.PlayerHistory.SteamID}}

{{if .PlayerHistory.Data }} <!-- This checks that the array is not empty -->
	{{range $entry := .PlayerHistory.Data}}
		Time: {{$entry.Time}} TotalKills: {{$entry.TotalKills}} <br/>
	{{end}}
{{end}}

```

**Comment by pinpox** [comment]
Removed for now
